HOW_WE_USE_GITHUB.md

```diff
@@ -1,7 +1,44 @@
-## The Issue Sorting Process at `conda`
+<!-- absolute URLs -->
+[conda-org]: https://github.com/conda
+[sub-team]: https://github.com/conda-incubator/governance#sub-teams
 
-> **Note:**
-> This process is similar to that of "triaging", but we've chosen to use different terminology because "triaging" is a word related to very weighty topics (*i.e.*, injuries and war) and we would like to be sensitive to those connotations. Additionally, we are taking a more "fuzzy" approach to sorting (*e.g.*, severities may not be assigned, etc.).
+[project-planning]: https://github.com/orgs/conda/projects/2/views/11
+[project-sorting]: https://github.com/orgs/conda/projects/2/views/11
+[project-support]: https://github.com/orgs/conda/projects/2/views/12
+[project-backlog]: https://github.com/orgs/conda/projects/2/views/13
+[project-sprint]: https://github.com/orgs/conda/projects/2/views/14
+
+[docs-toc]: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/
+[docs-actions]: https://docs.github.com/en/actions
+[docs-saved-reply]: https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply
+
+[workflow-sync]: https://github.com/conda/infra/blob/main/.github/workflows/sync.yml
+[labels-global]: https://github.com/conda/infra/blob/main/.github/global.yml
+
+<!-- relative URLs -->
+[workflow-issues]: /.github/workflows/issues.yml
+[workflow-project]: /.github/workflows/project.yml
+[labels-local]: /.github/labels.yml
+
+## How we use GitHub
```

Comment:
```suggestion
## How We Use GitHub
```

Total nitpick (changing the heading here to be capitalized like the other major headers in this document)